modules/ossm-threescale-external-service-entry-object.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-external-service-entry-object_{context}"]
+= The 3scale external `ServiceEntry` object
+
+To have the `threescale-wasm-auth` module authorize requests against 3scale, the module must have access to 3scale services. You can accomplish this within {ProductName} and Istio by applying an external `ServiceEntry` object.
+
+The custom resources set up the service entries for access from within {ProductShortName} to 3scale Hosted (SaaS) for the backend and system components of the Service Management API and the Account Management API. The Service Management API receives queries for the authorization status of each request. The Account Management API provides API management configuration settings for your services.
+
+Use the `oc apply` command to apply the following custom resource objects to your cluster, for example:
+
+* To save it to a file, use:
+
```

Comment:
Need a `+` continuation here to keep the code block below attached to the bullet.

Here and line 22.